pkg/synthetictests/duplicated_events.go

```diff
@@ -539,6 +540,33 @@ func isConsoleReadinessDuringInstallation(monitorEvent monitorapi.EventInterval,
 	return isEventDuringInstallation(monitorEvent, kubeClientConfig, regExp)
 }
 
+// isConfigOperatorReadinessFailed returns true if the event matches a readinessFailed error that timed out
+// in the openshift-config-operator.
+// like this:
+// ...ReadinessFailed Get \"https://10.130.0.16:8443/healthz\": net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)"
+func isConfigOperatorReadinessFailed(monitorEvent monitorapi.EventInterval, _ *rest.Config, _ int) (bool, error) {
+	regExp := regexp.MustCompile(probeTimeoutMessageRegExpStr)
```

Comment:
Probably not a big deal but this makes me wonder if the compile happens each time a call is made and would it be better outside the api?

Reply:
From a performance perspective, yes.  But I wanted to be explicit about the scope of that variable.

The compiler should be smart enough to know `probeTimeoutMessageRegExpStr` is static thus making that entire statement static, hence making it eligible to be extracted and run only once and outside the function.